### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by anuragpaliwal80**
### What problem does this PR solve?

Remove dead code

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by AsterNighT** [comment]
Uhh I'm not sure this is useless. @YangKeao would you take a look?

**Comment by YangKeao** [comment]
> Uhh I'm not sure this is useless. @YangKeao would you take a look?

I think a better way to fix this dead code is to "make it alive" :laughing: .

https://github.com/chaos-mesh/chaos-mesh/blob/2a38fd1eacec4c16619d140ae86e2bb4f303cac1/pkg/apiserver/common/common.go#L398-L402

These check should be

```go
if roleType == roleManager {
	verbs = `"get", "list", "watch", "create", "delete", "patch", "update"`
} else if roleType == roleViewer {
	verbs = `"get", "list", "watch"`
} else {
	return an error to the frontend
}
```

@fewdan What's your opinion?

File: pkg/apiserver/common/common.go [hunk 1]
```diff
@@ -397,8 +397,13 @@ func (s *Service) getRbacConfig(c *gin.Context) {
 	}
 	if roleType == roleManager {
 		verbs = `"get", "list", "watch", "create", "delete", "patch", "update"`
-	} else {
+	} else if roleType == roleViewer {
 		verbs = `"get", "list", "watch"`
+	} else {
+		c.Status(http.StatusInternalServerError)
+		_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(fmt.Errorf("roleType is neither manager nor viewer")))
```
Comment (STRRL) [inline pkg/apiserver/common/common.go:404]: How about using `http.BadRequest` and `utils.ErrInvalidRequest` instead?

**Comment by fewdan** [comment]
It seems that you should fix DCO first?

**Comment by anuragpaliwal80** [comment]
Not sure what's happening with DCO, I tried both of its commands still no luck.
```
git commit --amend --no-edit --signoff
git push --force-with-lease origin ap_8
```

**Comment by anuragpaliwal80** [comment]
Looks like with `git rebase HEAD~7 --signoff` it got fixed.

**Review by AsterNighT — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**